.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on: 
+  push:
+    tags:
+      - '!player/'
+      - 'v*'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up node.js
+        uses: actions/setup-node@v3.3.0
+        with:
+          node-version-file: .nvmrc
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Test
+        run: npm test
+
+      - name: Build
+        run: gulp build-prod
```

Comment:
You can also `reuse` other workflows: 

https://docs.github.com/de/actions/using-workflows/reusing-workflows

Because this is copied. So what would be my suggestion is to use `upload` and `download` artifact to pass over the build artifact to the release artifact.

Reply:
Agreed, updated (over several commits)